haddock.cabal

```diff
@@ -12,9 +12,10 @@ bug-reports:          https://github.com/haskell/haddock/issues
 copyright:            (c) Simon Marlow, David Waern
 category:             Documentation
 build-type:           Simple
-cabal-version:        >= 1.10
+cabal-version:        >= 2.0
```

Comment:
We'd need to check whether GHC HEAD can still be bootstrapped with this (GHC's build uses only the  `haddock.cabal` file). Would `buildtools: haddock` work as well?